source/fundamentals/authentication/enterprise-mechanisms.txt

```diff
@@ -33,6 +33,14 @@ parameters of the
   parameter if using a value other than ``mongodb``
 - Specify a ``SERVICE_REALM`` value in the ``authMechanismProperties``
```

Comment:
S: All items in this list are imperative and [therefore should end with periods.](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items) Don't have to change it in this ticket if you think it's outside the scope.